docs/internals/contributing/writing-documentation.txt

```diff
@@ -45,11 +45,16 @@ documentation is transformed into HTML, PDF, and any other output format.
 __ https://www.sphinx-doc.org/
 __ https://docutils.sourceforge.io/
 
-To build the documentation locally, install Sphinx:
+To build the documentation locally, install the doc requirements from the
+project root
 
 .. console::
 
-     $ python -m pip install Sphinx
+     $ python -m pip install -r docs/requirements.txt
+
+It's necessary to run this command from the project root because the docs are
```

Comment:
This parenthetical note (since "from the project root" was already mentioned), could be enclosed in parentheses.